renderdoc/driver/metal/metal_resources.h

```diff
@@ -124,23 +124,29 @@ inline MTL::Resource *Unwrap(WrappedMTLResource *obj)
   return Unwrap<MTL::Resource *>((WrappedMTLObject *)obj);
 }
 
+enum MetalCmdBufferStatus
```

Comment:
Prefer using `enum class` for internal enums like this.

There are defines `BITMASK_OPERATORS` and `ITERABLE_OPERATORS` you can use for bitsets like this to ensure the enum class can still easily do bitwise operators.